.github/workflows/docs-ci.yml

```diff
@@ -38,6 +38,13 @@ jobs:
         run: |
           .github/scripts/gradle-properties.sh >> gradle.properties
           cat gradle.properties
+          
+      - name: Set DHC Version
+        id: dhc-version
+        run: |
+          ./gradlew printVersion -q 
+          ./gradlew printVersion -q > dhc_version.txt
```

Comment:
We've got an `outputVersion` task we can use instead; it can be invoked during the respective `burrunan/gradle-cache-action@v1` step. ie:

```
arguments: --scan combined-javadoc:allJavadoc outputVersion
```

There's an example of this in build-ci.yml. (Ditto for the other places you are using printVersion.)